adapters/repos/db/inverted/bm25_searcher.go

```diff
@@ -38,6 +37,7 @@ import (
 	"github.com/weaviate/weaviate/entities/schema"
 	"github.com/weaviate/weaviate/entities/searchparams"
 	"github.com/weaviate/weaviate/entities/storobj"
+	"golang.org/x/sync/errgroup"
```

Comment:
can this file be reverted to it's original form? those changes introduced here are only empty spaces and a change in import statement. Those changes made here doesn't bring anything new to the project.